Library/Formula/qt.rb

```diff
@@ -25,6 +25,9 @@ class Qt < Formula
     sha256 "c8a0fa819c8012a7cb70e902abb7133fc05235881ce230235d93719c47650c4e"
   end
 
+  # Make `HOMEBREW_PREFIX/lib/qt4/plugins` an additional plugin search path.
+  patch :DATA
```

Comment:
Sure, we just don't want to have long-running patches that will never be accepted; that's a long-time policy of Homebrew's. Editing installed files, adding symlinks or wrapper scripts is preferable but doing actual binary patching is another smell.